### DISCUSSION TITLE
Why was `Microsoft.PowerShell.UnixTabCompletion` archived?

### DISCUSSION THREAD

**Discussion by jpawlowski** [PowerShell/PowerShell#24174] [Q&A] [answered]
Hi team,

I noticed that [`Microsoft.PowerShell.UnixTabCompletion`](https://github.com/PowerShell/UnixCompleters) was archived back in June 2024.

I was wondering if this was intentional, or if this was by mistake?

I am asking because one can read from [#20 Deprecate Microsoft.PowerShell.UnixCompleters](https://github.com/PowerShell/UnixCompleters/issues/20) that there was the intention to clean up a bit with the overlapping / old `Microsoft.PowerShell.UnixCompleters` module. Since the GitHub repository of the newer module is named after the old module, it seems to me we could expect a new repository to appear under the name [`github.com/powershell/UnixTabCompletion`](https://github.com/powershell/UnixTabCompletion) that doesn't exist yet?

Could something have mixed up here, or is there a successor in the works?
Should we no longer use `Microsoft.PowerShell.UnixTabCompletion` then?

I'm looking forward to getting some light on this ;-)

Many thanks,
—Julian

**Comment by ThomasNieto** [comment, marked as answer]
You'll notice a lot of modules were archived recently. It's a new policy that modules not being actively developed are archived. I had them add a note to the Archive module.

If in the future they devote resources it would be unarchived.

**Reply by jpawlowski** [reply]
That makes sense. Even if (temporarily) archiving a repository also sets all open issues and pull requests to read only, there is a better indicator and explanation for why issues and pull requests are not answered. Even if you can always guess the reason, there is always a little bit of hope with expectations like “maybe someone will have time at Christmas”.

The new policy makes perfect sense to me. It's just difficult to find out the “why” better.

So the bottom line should be: Keep doing what you're doing if it works for you.

That being said: If there was a process for community members to take over the maintenance work for smaller projects, that might be helpful. Sometimes it's just small things that need to be fixed, and rather than forking the whole project and renaming it “yet another module” on PSGallery, I'd rather see this small fix in the existing module.

**Reply by ThomasNieto** [reply]
Here is where it's addressed in the community call.

https://youtu.be/TPf4FqlWaGA?feature=shared&t=1574
